code/game/objects/items/rogueweapons/melee/swords.dm

```diff
@@ -615,3 +615,12 @@
 	smelt_bar_num = 2
 	max_blade_int = 150
 	max_integrity = 200
+
+/obj/item/rogueweapon/sword/long/blackflamb
+	force = 20
+	force_wielded = 32
+	icon_state = "blackflamb"
+	name = "blacksteel flamberge"
+	desc = "A strange sword with a winding blade forged of blacksteel and a rontz pommel."
+	smeltresult = /obj/item/ingot/blacksteel
+	max_integrity = 200
```

Comment:
```suggestion
	max_integrity = 200
	sellprice = 220
```
Decorative sword. Likely wasteful to sell or treasure hoard it but still
Rontz = 100 
Blacksteel bar = 90
Lil extra sprinkled in for labor

Reply:
I avoided adding a higher value because it would bloat the treasury even more, when at it's core it is a weapon. Terminus Est doesn't have a high value either, as an example. Later on the value could possibly be tweaked.